### ISSUE TITLE
WBFS_OPEN:-01 error when starting WiiFlow

### ISSUE THREAD

**Issue by GoogleCodeExporter** [WhiteNeo/wiiflow#106]
```
What steps will reproduce the problem?
1.When I startup WiiFlow I get WBFS_OPEN:-01 error
2.
3.

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?
WiiFlow 1.1 with OS Wiimenu 4.1E

Please provide any additional information below.

```

Original issue reported on code.google.com by `marcovan...@lubixeurope.com` on 17 Jan 2010 at 8:18

**Comment by GoogleCodeExporter** [comment]
```
This happened to me a long time ago, this is a problem that appears when your 
wbfs 
partition is not the closer to the C, I explain, when I had wbfs and this 
problem 
presented me the wbfs partition was the H and my other partition where the G so 
I 
changed them and wiiflow worked again.
```

Original comment by `sanchiag...@hotmail.com` on 17 Jan 2010 at 6:57

**Comment by GoogleCodeExporter** [comment]
```
Whats the config on your HD?
Full partition WBFS?
```

Original comment by `nfernan...@gmail.com` on 20 Jan 2010 at 7:34

**Comment by GoogleCodeExporter** [comment]
```
i have the same problem......but mine is half fat32 the half wbfs, i keep 
getting 
that error if i try to use wiiflow to load the hdd games, but when i use 
uloader it 
works perfectly. i just want to be able to use wiiflow cuz it looks much nicer 
and u 
can have your own BG music  anyone can help us??? that b great, thanks
```

Original comment by `tam...@gmail.com` on 4 Feb 2010 at 7:14

**Comment by GoogleCodeExporter** [comment]
```
my wbfs is closer to C but it still doesn't work. 
```

Original comment by `tam...@gmail.com` on 4 Feb 2010 at 7:16

**Comment by GoogleCodeExporter** [comment]
```
What revision you guys tried ?
```

Original comment by `thedarkn...@gmail.com` on 7 Feb 2010 at 1:06

**Comment by GoogleCodeExporter** [comment]
```
rev17
```

Original comment by `tam...@gmail.com` on 9 Feb 2010 at 3:10

**Comment by GoogleCodeExporter** [comment]
```
I got the same problem yesterday.

My HDD has two partitions. The first is FAT32, the second WBFS.

Has anyone an idea?
```

Original comment by `sireni...@gmail.com` on 16 Feb 2010 at 7:40

**Comment by GoogleCodeExporter** [comment]
```
I get same issue when using R31, I have a 500GB drive with a 4Gb FAT partition, 
and
the rest is WBFS.

Plasma
```

Original comment by `pla...@gmx.co.uk` on 8 Mar 2010 at 12:19

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
-Try to make the WBFS as the first partition and the fat32 the second one...
-Delete the wiiflow.ini on wiiflow folder
```

Original comment by `naxox2...@gmail.com` on 8 Mar 2010 at 2:40

**Comment by GoogleCodeExporter** [comment]
```
I have the same issue.
4.2U
HHD Seagate 250gb WBFS unique partition.
WBFS_OPEN:-01 error
```

Original comment by `wsanriv` on 9 Mar 2010 at 6:20

**Comment by GoogleCodeExporter** [comment]
```
I had this problem too. My disc was partitioned with first a FAT32, then a WBFS 
and
last a NTFS partition. 
This configuration has work on the two wii's a modified before. But for some 
reason
it didn't work for this disk. 
So I changed the order of the partitions to WBFS-FAT32-NTFS and then it worked!

(WiiFlow 1.1 with OS Wiimenu 4.1E)
```

Original comment by `hrung...@gmail.com` on 11 Mar 2010 at 9:52

**Comment by GoogleCodeExporter** [comment]
```
I have this issue with r43 on a FAT32 only partition. I have the .wbfs files as 
such 
usb://wbfs/gamefolder/game.wbfs

Also does it to me if i try a NTFS drive with the apps folder on the SD.
```

Original comment by `k.krst...@gmail.com` on 5 Jun 2010 at 8:28

**Comment by GoogleCodeExporter** [comment]
```
I fixed this issue by deleting the wiiflow created wiiflow.ini [which was 
corrupted]. 
Editing my old .ini i had saved from when i was using WBFS, to say 
'partition=FAT1'.

Works now.
```

Original comment by `k.krst...@gmail.com` on 5 Jun 2010 at 8:59

**Comment by GoogleCodeExporter** [comment]
```
Fixed in wiiflow 2.0
```

Original comment by `Miigotu` on 19 Jun 2010 at 10:32
- Changed state: **Fixed**
